makejdk-any-platform.1

```diff
@@ -137,6 +141,11 @@ specify any custom user make arguments.
 .BR \-\-no\-adopt\-patches
 This indicates to the build script that the build does not have Temurin's additional patches applied.
 .TP
+.BR  \-o ", " \-\-openjdk\-source " " \fI<dir/archive>\fR
```

Comment:
Its not minor. Its quite  major think.  if -l would be used, than it would be first short hand, which is letter, which is not used in the long one. The shortcut is critical. Just image writing the long one more then once per day. You would need to do a wrapper for that.  So to make you happy,  I would go with  \-l ", " \-\-openjdk\-source-location "

How much do you want to chage this? Switch is api, Unused by pipelnes, but still api.

Reply:
> How much do you want to chage this? Switch is api, Unused by pipelnes, but still api.

I'd say 20-30% - certainly not something I'd block on if there are strong opposing feelings.

Reply:
I agree wit -o being confused by "output. Will try to  repalce it by ` "-l ", " --openjdk-source-location"`